script.js

```diff
@@ -22,22 +23,32 @@ function addBook() {
         number_of_copies: number_of_copies
     }
     //books.push('book');
-    //console.log(`${title} = ` + 'the newbook is table');
+   
     let hasBook = false;
     let bookSave = document.getElementById('book-save');
     let bookBd = document.getElementById('book-bd');
-
+    
     for(let i = 0; i < books.length; i++) {
         let currentBook = books[i];
         if(book.id_number == currentBook.id_number) {
             hasBook = true;
+            books.push(book);
             break;
         }
     }
+    //for(let i = 0; i < books.length; i++) {
+        if(book.id_number == books.id_number) {
```

Comment:
if we don't have this book in database , then we need to save it

Reply:
create function to show Error Status
- show block with error message
- hide block with success message
- clear input